app/models/iam_user_identity.rb

```diff
@@ -44,7 +44,8 @@ def self.build_from_iam_profile(iam_profile)
       iam_mhv_id: valid_mhv_id(iam_profile[:fediam_mhv_ien]),
       last_name: iam_profile[:family_name],
       loa: { current: loa_level, highest: loa_level },
-      middle_name: iam_profile[:middle_name]
+      middle_name: iam_profile[:middle_name],
+      sign_in: { service_name: "oauth_#{iam_auth_n_type}", account_type: iam_profile[:fediamassur_level] }
```

Comment:
This is somewhat unrelated but will populate the credential and LOA tags in Sentry for mobile instead of having them say "not-logged-in"